assets/cellrangermulti_samplesheet.csv

```diff
@@ -0,0 +1,9 @@
+sample,fastq_1,fastq_2,feature_type,protocol,expected_cells
```

Comment:
I think we can change. The only reason I added as `feature_type` because the same sample, can have data of different feature types. So, would not be the type for the whole sample.

But, code-wise, would mean just changing the value.

What you guys prefer?